package.json

```diff
@@ -8,6 +8,7 @@
     "start": "npm run test:base -- --watch --growl",
     "test:base": "mocha \"tests/lib/**/*.js\" --reporter dot",
     "test": "nyc npm run test:base -- \"tests/integrations/*.js\" --timeout 60000",
+    "test:define-macros-order": "mocha \"tests/lib/rules/define-macros-order.js\" --timeout 60000",
```

Comment:
This is fine for testing locally, but please avoid committing and pushing debugging changes like this. I'll revert this change.

Reply:
Reverted in [`efd97d7`](https://github.com/vuejs/eslint-plugin-vue/pull/2501/commits/efd97d7d564f8d8c1a1c0eba38d175fd5c49cbda).